prime-dotnet-webapi/Services/EnrolleeService.cs

```diff
@@ -903,5 +904,28 @@ public async Task<IEnumerable<int>> GetNotifiedEnrolleeIdsForAdminAsync(ClaimsPr
                 .Select(en => en.EnrolleeId)
                 .ToListAsync();
         }
+
+        public async Task<IEnumerable<string>> GetEnrolleeEmails(BulkEmailType bulkEmailType)
+        {
+            Expression<Func<Enrollee, bool>> predicate = bulkEmailType switch
+            {
+                BulkEmailType.CommunityPractice => e => e.EnrolleeCareSettings.Any(cs => cs.CareSettingCode == (int)CareSettingType.CommunityPractice),
+                BulkEmailType.CommunityPharmacy => e => e.EnrolleeCareSettings.Any(cs => cs.CareSettingCode == (int)CareSettingType.CommunityPharmacy),
+                BulkEmailType.HealthAuthority => e => e.EnrolleeCareSettings.Any(cs => cs.CareSettingCode == (int)CareSettingType.CommunityPharmacy),
```

Comment:
Should it be `cs => cs.CareSettingCode == (int)CareSettingType.HealthAuthority` ?